x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema.tsx

```diff
@@ -43,7 +41,6 @@ import { SchemaLogic } from './schema_logic';
 export const Schema: React.FC = () => {
   const {
     initializeSchema,
-    onIndexingComplete,
```

Comment:
@scottybollinger I wasn't sure whether to also delete the action from the logic file or not - I left it alone in this PR because with the schema/indexing changes I'm not really sure what all needs to change in that logic file and what doesn't.

Reply:
It looks like just [these lines](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.ts#L181-L187) and the [action](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.ts#L53) and interface [prop](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.ts#L129), along with the [test](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema_logic.test.ts#L143-L153). I can do it in another PR if you like.

Reply:
It would be awesome if you could (apologies for the imposition)! I was worried there might be more to potentially remove that I might've missed and was hoping you could grab that 🙏 Thanks Scotty!